src/localization/bundles/en.json

```diff
@@ -714,9 +716,15 @@
+    "server.addserver.installos.cancel.confirm": "This will cancel the OS installation for the new compute server(s).  \n\nDo you want to cancel?",
+    "server.replace.compute.osinstall.failure.cancel.confirm": "Cancelling wil close the window and any error messages will no longer be available to view. \nAfter cancel, the new and undeployed compute server {0} shows up in servers Summary page, go to Add server page and unassign it from the role before replace compute server {1} again.  \n\nDo you want to cancel?",
+    "server.replace.compute.failure.prepadd.cancel.confirm": "Cancelling will close the window, and any error messages will no longer be available to view. \n\nThe following steps need to be performed manually on the cloud deployer before clicking Yes to cancel... \n1. cd ~/openstack/my_cloud/definition/data \n2. Remove the new compute server {0} from servers.yml \n3. git commit -a -m 'commit message' \n4. cd ~/openstack/ardana/ansible \n5. ansible-playbook -i hosts/localhost config-processor-run.yml \n6. ansible-playbook -i hosts/localhost ready-deployment.yml  \n\n If the Summary page continues to display the removed server, refresh the page using the web browser.  \n\n Do you want to cancel?",
+    "server.replace.compute.failure.add.cancel.confirm": "Cancelling could potentially leave the process of adding the new compute server {0} in a half state. It will close the window, and any error messages will no longer be available to view. \n\nThe following steps need to be performed manually on the cloud deployer if failed deploying the new server {0} before clicking Yes to cancel... \n1. cd ~/openstack/my_cloud/definition/data \n2. Remove the new compute server {0} from servers.yml \n3. git commit -a -m 'commit message' \n4. cd ~/openstack/ardana/ansible \n5. ansible-playbook -i hosts/localhost config-processor-run.yml \n6. ansible-playbook -i hosts/localhost ready-deployment.yml  \n\n If the Summary page continues to display the removed server, refresh the page using the web browser.\n\n Do you want to cancel?",
+    "server.replace.compute.failure.delete.cancel.confirm": "Cancelling could potentially leave the process of deleting the old compute server {0} in a half state. It will close the window, and any error messages will no longer be available to view.\n\nDo you want to cancel?",
+    "server.replace.compute.failure.disable.cancel.confirm": "Cancelling could potentially leave the process of disabling the old compute server {0} in a half state. It will close the window, and any error messages will no longer be available to view.\n\nDo you want to cancel?",
```

Comment:
I'm not sure that the meaning of "in a half state" is clear here... would be curious about alternatives "partially complete" or "in an incomplete state" ... I dont know...

Reply:
What I try to say here is when it failed in the middle of running deployment,  for example site.xml --limite...depending where it failed,  instead of fixing the problems and retry, click cancel can potentially leave the cloud in a wired situation.